nilearn/_utils/helpers.py

```diff
@@ -145,3 +146,21 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
         return wrapper
     return _remove_params
+
+
+def _stringify_path(path):
```

Comment:
not sure it would need to start with `_` as it is in the internal `_utils` module, and meant to be used outside of it. however function names in nilearn are not consistent in that regard so we can keep it like this for now and maybe consider clarifying this in another pr

Reply:
I will change it here since it is quick but I agree on opening an issue/pr to standardize function naming.